packages/perseus/src/widgets/interactive-graphs/mafs-graph.tsx

```diff
@@ -70,79 +72,106 @@ export const MafsGraph = (props: MafsGraphProps) => {
             }}
         >
             <View
-                className="mafs-graph"
                 style={{
-                    width,
-                    height,
-                    position: "relative",
-                    padding: "25px 25px 0 0",
-                    boxSizing: "content-box",
-                    marginLeft: "20px",
-                    marginBottom: "20px",
-                    pointerEvents: props.static ? "none" : "auto",
-                    userSelect: "none",
+                    width: width,
                 }}
             >
-                <LegacyGrid
-                    box={props.box}
-                    backgroundImage={props.backgroundImage}
-                />
                 <View
+                    className="mafs-graph"
                     style={{
-                        position: "absolute",
-                        bottom: 0,
-                        left: 0,
+                        height,
+                        width,
```

Comment:
Do we need both this `width` and the one on the wrapper `View`, above? It seems like maybe we could get away with just one...